viewer/.env

```
@@ -3,3 +3,4 @@
 VUE_APP_I18N_LOCALE=en
 VUE_APP_I18N_FALLBACK_LOCALE=en
 VUE_APP_DATA_URL=./
+VUE_APP_FULLWIDTH_LIMIT=1500
```

```suggestion
VUE_APP_FULLWIDTH_LIMIT=1216
```

Let's stay consistent with one of [Bulma's breakpoints](https://bulma.io/documentation/overview/responsiveness/#breakpoints).